src/clj/game/cards/operations.clj

```diff
@@ -615,7 +615,18 @@
                               :trace {:base 2 :msg "give the Runner 1 tag"
                                       :delayed-completion true
                                       :effect (effect (tag-runner :runner eid 1))}}}}
-
+   
+   "MCA Informant"
+   {:req (req (some #(has-subtype? % "Connection") (all-installed state %))) 
```

Comment:
`(all-installed state %)` doesn't work here. Would need to be `(concat (all-installed state :runner) (all-installed state :corp))`